Sources/Sentry/Public/SentryTransactionContext.h

```diff
@@ -71,6 +87,23 @@ SENTRY_NO_INIT
                 parentSpanId:(nullable SentrySpanId *)parentSpanId
                parentSampled:(SentrySampleDecision)parentSampled;
 
+/**
```

Comment:
Same as in https://github.com/getsentry/sentry-cocoa/pull/4751#discussion_r1930846919